dpnp/dpnp_iface_indexing.py

```diff
@@ -409,6 +418,15 @@ def put_along_axis(x1, indices, values, axis):
         else:
             return dpnp_put_along_axis(x1_desc, indices_desc, values_desc, axis)
 
+    if config.__DPNP_OUTPUT_DPCTL__:
```

Comment:
Are you going to put such code into any python level interface functions?
move it to call_origin()